addons/html_editor/static/tests/data-oe/protected.test.js

```diff
@@ -212,3 +212,28 @@ test("should protect disconnected nodes", async () => {
         `<div data-oe-protected="true"></div>`
     );
 });
+
+test("should not crash when changing attributes and removing a protecting anchor", async () => {
+    let historyPlugin;
+    patchWithCleanup(HistoryPlugin.prototype, {
```

Comment:
no big deal, but maybe we should have in the future a nicer way to get references to plugin. maybe
```js
const { editor, el, plugins } = await setupEditor(...);
const historyPlugin = plugins.get("history");
```